queries.sql

```diff
@@ -121,4 +121,52 @@ SELECT species, AVG(escape_attemps) AS "Average Escape Attempts"
  FROM animals
  WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
  GROUP BY species
- HAVING AVG(escape_attemps) IS NOT NULL;
\ No newline at end of file
+ HAVING AVG(escape_attemps) IS NOT NULL;
+
+ /* Answers using JOIN */
+ --what animals belong to Melody Pond?
+ SELECT animals.name AS animal_name
+ FROM animals
+ JOIN owners ON animals.owner_id = owners.id
+ WHERE owners.full_name = 'Melody Pond';
+
+ /*list of all animals that are pokemon (their type is Pokemon)*/
+SELECT animals.name AS animal_name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```

Comment:
- [ ] When i run and test your sql queries, this particularly highlighted query return 5 values as shown in the image below however your screenshot shows only four, `Ditto` is missing. Please kindly check your queries and correct the result of the querires :+1: 
![5](https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/89055502/7ff02b77-646f-4b48-af47-b72eeac308b5)

Reply:
Hello @Efiamotu-1 , 
Thanks for your explanation ,I appreciate it,
I already modified what you suggested,
here is the screenshot where Ditto appears.
<img width="297" alt="DittoIncluded" src="https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/22691013/8d969bb0-2f59-455c-b84a-f06a68cb9ef8">